components/Navbar/Navbar.jsx

```diff
@@ -54,6 +54,7 @@ const Navbar = () => {
       {/* Nav Links */}
       <ul data-nav-toggle={navToggle} className="animation">
         <button
+          aria-label="theme toogle"
```

Comment:
There is a spelling mistake, Can we correct it?
```diff
- toogle
+ toggle
```

Reply:
Yeah, I would!